modin/pandas/groupby.py

```diff
@@ -1297,6 +1297,31 @@ def _iter(self):
                 for k in (sorted(group_ids) if self._sort else group_ids)
             )
 
+    def unique(self):
+        return self._check_index(
+            self._wrap_aggregation(
+                type(self._query_compiler).groupby_unique,
+                numeric_only=False,
+            )
+        )
+
+    def nlargest(self, n=5, keep="first"):
+        return self._check_index(
+            self._wrap_aggregation(
+                type(self._query_compiler).groupby_nlargest,
+                agg_kwargs=dict(n=n, keep=keep),
+                numeric_only=True,
```

Comment:
does numeric_only consider booleans as well? It looks like bool dtypes work for `nlargest` and `nsmallest`.

Reply:
Hmm so this should be `False` then?

Reply:
@naren-ponder it looks like bool is considered as numeric values. Can you just double check this with other dtypes? Looks like strings aren't supported, so I think this is ok.